client/src/views/page-components/account/history/history.ts

```diff
@@ -0,0 +1,99 @@
+import { $ } from '@src/utils/helper';
+import { createDOMWithSelector, matchCategoryAndImg } from '@src/utils/helper';
+import './history.scss';
+
+export default class History {
+  state: any;
+  history: HTMLElement;
+  constructor({ parent, state }) {
+    this.history = createDOMWithSelector('div', '.history');
+
+    parent.appendChild(this.history);
+    this.setProperty(state);
+    this.render();
+  }
+
+  setProperty(state): void {
+    this.state = state;
+  }
+
+  render(): void {
+    $('.history').innerHTML = `
+        ${this.createHistoryHeader()}
+        ${this.createHistoryContent()}
+        `;
+  }
+
+  createHistoryHeader(): string {
+    return `
+      <div class='history__header'>
+        <div class='history__header-left'>
+          <span class='history__text'>내역</span>
+          <span class='history__add'>+</span>
+        </div>
+        <div class='history__header-right'>
+          <span>수입</span>  
+          <span>${this.state.income}</span>
+          <span>지출</span>
+          <span>${this.state.expenditure}</span>
+        </div>
+      </div>`;
+  }
+
+  createHistoryContent(): string {
+    return `
+      <table class='table'>
```

Comment:
className에서 어떤 테이블인지 명확히 파악 되지 않는 것 같습니다 ㅜ ㅜ
물론 저희 프로젝트에서 하나의 테이블만 쓰는 것 같아 문제가 되어 보이진 않지만...
table이라는 class name은 개인적으로 추상적인 느낌이네요
그리고 저희 history 계속 사용하는건가요?

```suggestion
      <table class='history__table'>
```
```suggestion
      <table class='account__table'>
```

Reply:
className 좋은지적이네요! 다만  history를 원래 페이지 이름으로 안쓰기로 했었는데 그래서 account로 결정이 났는데 음 내부에 내역을 나타낼만한 단어가 history가 생각이 나고 여기 자체에 클래스로만 이름을 지어주니 크게 헷갈릴부분이 없을거같아서 이렇게 했습니다!